upgrading/upgrade-roxctl.adoc

```diff
@@ -17,6 +17,12 @@ To upgrade {product-title} to the latest version, you must perform the following
 * Upgrade Scanner
 * Verify that all secured clusters are upgraded
 
+[IMPORTANT]
+====
+* In order to ensure optimal functionality it is recommended to keep your secured clusters up to date and on the same version as Central. Out of date Sensor versions are supported for 6 months, but may not have the full functionality of an up to date version.
+* Enabling automated upgrades for your secured clusters is strongly recommended. Should you prefer manually upgrading your secured clusters refer to the xref:upgrade-roxctl.adoc#upgrade-secured-clusters[manual on upgrading secured clusters].
```

Comment:
I believe the section below (the one with the title `== Upgrading all secured clusters` goes quite extensively about automatic upgrades, therefore it isn't necessary to call that out additionally.
```suggestion
```